roles/lib_openshift/src/doc/service

```diff
@@ -0,0 +1,121 @@
+# flake8: noqa
+# pylint: skip-file
+
+DOCUMENTATION = '''
+---
+module: oc_service
+short_description: Create, modify, and idempotently manage openshift services.
+description:
+  - Manage openshift service objects programmatically.
+options:
+  state:
+    description:
+    - State represents whether to create, modify, delete, or list
+    required: true
```

Comment:
Options with `default`s set do not need `required` set to `true`. For example, look at `kubeconfig` below.